service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentUploadHelperTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.fpl.utils.DocumentUploadHelper;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class DocumentUploadHelperTest {
+    private static final String AUTH_TOKEN = "token";
+    private static final String USER_ID = "1";
+
+    @Mock
+    private IdamClient idamClient;
+
+    @Mock
+    private RequestData requestData;
+
+    @InjectMocks
+    private DocumentUploadHelper documentUploadHelper;
+
+    @BeforeEach
+    void setUp() {
+        when(requestData.authorisation()).thenReturn(AUTH_TOKEN);
+    }
+
+    @Test
+    void shouldReturnUploadedDocumentUserInfoForUserWithHmctsRole() {
```

Comment:
Don't like info, info is too vague in my opinion. What about `shouldReturnUploadedDocumentUserNameForUserWithHmctsRole`. Same with other tests.

Reply:
'HMCTS' is not userName. I will leave it unless you have other suggestions.

Reply:
No it's not, it is a role. So if you expect the method to a return a role then `shouldReturnUploadedDocumentUserRoleForUserWithHmctsRole` and in other test that returns email `shouldReturnUploadedDocumentUserEmailForUserWithNonHmctsRole`.